x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/per_field_rule_diff_tab.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo } from 'react';
+import type { PartialRuleDiff, RuleFieldsDiff } from '../../../../../common/api/detection_engine';
+import { getFormattedFieldDiff } from '../../logic/rule_details/get_formatted_field_diff';
+import { UPGRADE_FIELD_ORDER } from './constants';
+import { RuleDiffHeaderBar, RuleDiffSection } from './diff_components';
+import { getSectionedFieldDiffs } from './helpers';
+import type { RuleFieldDiff } from '../../model/rule_details/rule_field_diff';
+
+interface PerFieldRuleDiffTabProps {
+  ruleDiff: PartialRuleDiff;
+}
+
+export const PerFieldRuleDiffTab = ({ ruleDiff }: PerFieldRuleDiffTabProps) => {
+  console.log(ruleDiff);
+
+  const fieldsToRender = useMemo(() => {
+    const fields: RuleFieldDiff[] = [];
+    for (const field in ruleDiff.fields) {
+      if (Object.hasOwn(ruleDiff.fields, field)) {
+        const typedField = field as keyof RuleFieldsDiff;
+        const formattedDiffs = getFormattedFieldDiff(typedField, ruleDiff.fields);
+        fields.push({ formattedDiffs, fieldName: typedField });
```

Comment:
We're going to need a dictionary to translate field names as they exist in the rule schema to their UI-version. For example, `note` is actually `Investigation guide`. I guess we can create a dictionary only for the one that are completely different, and for the others, not found in the dict, just take the field name and camel case it as you are doing now.

Reply:
Started in `x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/diff_components/translations.ts`